Analysis/IsolatedLeptonFinder/include/IsolatedLeptonFinderProcessor.h

```diff
@@ -31,103 +33,145 @@ class IsolatedLeptonFinderProcessor : public Processor {
 		virtual Processor*  newProcessor() { return new IsolatedLeptonFinderProcessor ; }
 
 		IsolatedLeptonFinderProcessor() ;
+		IsolatedLeptonFinderProcessor(const IsolatedLeptonFinderProcessor &);
```

Comment:
Copy and assignment operator are never used so ` ... = delete;` both of them

Reply:
This is to fix this warning at compile time:
```shell
warning: 'class IsolatedLeptonFinderProcessor' has pointer data members
warning: but does not override 'IsolatedLeptonFinderProcessor(const IsolatedLeptonFinderProcessor&)
warning: or 'operator=(const IsolatedLeptonFinderProcessor&)'
```

As it is now, the IsolatedLeptonFinder has not a single compiler warning. Do you know another way to circumvent that? If the warning is senseless we should disable it in the compiler settings.

Reply:
I mean you can do
```c++
IsolatedLeptonFinderProcessor(const IsolatedLeptonFinderProcessor &) = delete;
```
to tell the compiler do remove this function

Reply:
That removes half the warning, this one remains:
```
warning: ‘class IsolatedLeptonFinderProcessor’ has pointer data members [-Weffc++]
 class IsolatedLeptonFinderProcessor : public Processor {
warning:   but does not override ‘operator=(const IsolatedLeptonFinderProcessor&)’ [-Weffc++]
```

Reply:
Done!